charts/move2kube/README.md

```diff
@@ -0,0 +1,21 @@
+Move2kube
+===========
+
+# Installation
+From `charts` folder run 
+```console
+helm install move2kube move2kube
+```
+
+Then update `move2kube_url` in [values file](values.yaml) with the route of the move2kube instance:
+```console
+$ oc -n sonataflow-infra get routes move2kube-route 
```

Comment:
I want the move2kube to be accessible from the outside; when the user receive a notification, it uses the move2kube URL provided in the values, with a svc it would not be accessible

Reply:
+1 for using the service

Reply:
how can a user access the service when (s)he receives the notification on backstage to fill the Q&A?

Reply:
If it goes into the email/notification then the route is needed. comment resolved

Reply:
wonder if the questionae link would be affected by that change ?

Reply:
well, the move2kubeURL in the values is the one given to the workflow and the workflow uses this value when sending the notification

Reply:
so when using the k8s service url it will. Then a user clicking the link wouldn't be able to resolve that.

Reply:
yep

Reply:
so unfortunately we can't use svc at the moment. maybe in the future we could refactor that urls usage